### DISCUSSION TITLE
Basic question: Background color of screens in <Tabs>

### DISCUSSION THREAD

**Discussion by stuontheblockchain** [expo/router#684] [Q&A]
```
    <Tabs >
      <Tabs.Screen name='home' options={{header: header}}/>
      <Tabs.Screen name='settings' options={{header: header}}/>
    </Tabs>
```
Given a setup like that, how can I set the default background color for different screens? Currently I have an element with height 100%, but I feel like there is certainly a better solution. All of the help online I have found relates to Tab.Navigator. Any help is appreciated greatly.

**Comment by wcastand** [comment]
```ts
<Tabs sceneContainerStyle={{ backgroundColor: color.white }} ... >
```

is what you are looking for?

**Reply by 0tii** [reply]
In my case this only changed the color of the header bar
